### PR TITLE
kdb plugin-check

### LINKED ISSUE #4878: 0.9.13 release error
We have an error for the plugins `timeofday` and `tracer` when running `kdb plugin-check`.

```
will try check the plugin timeofday
Plugin did not pass all checks:
See description below:
The symbol "commit" has the same value as another symbol!

[...]

will try check the plugin tracer
Plugin did not pass all checks:
See description below:
The symbol "commit" has the same value as another symbol!
```

It was caught on Homebrew builds but is reproducible on the Fedora packages as well.

### PR DESCRIPTION
**Pull request by kodebach**
Fix #4878 

## Basics

- [ ] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which contains `_(my name)_`)
      **Please always add them to the release notes.**
- [ ] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy. If not, fix **in this order**:
  - add a line in `doc/news/_preparation_next_release.md`
  - reformat the code with `scripts/dev/reformat-all`
  - make all unit tests pass
  - fix all memleaks
  - fix the CI itself (or rebase if already fixed)
- [ ] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation (see [Documentation Guidelines](https://www.libelektra.org/devgettingstarted/documentation))
- [ ] I fixed all affected decisions (see [Decision Process](https://www.libelektra.org/decisions/decision-process))
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://www.libelektra.org/devgettingstarted/coding))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned [every code](/.reuse/dep5) not directly written by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

- [ ] Add the "work in progress" label if you do not want the PR to be reviewed yet.
- [ ] Add the "ready to merge" label **if everything is done** and no further pushes are planned by you.

### CODE REVIEW

File: src/plugins/tracer/tracer.c [hunk 1]
```diff
@@ -120,6 +120,24 @@ int elektraTracerSet (Plugin * handle, KeySet * returned, Key * parentKey)
 	return nr_keys;
 }
 
+int elektraTracerCommit (Plugin * handle, KeySet * returned, Key * parentKey)
+{
+	ssize_t nr_keys = 0;
+	Key * k = 0;
+
+	printf ("tracer: set(%p, %s, %s): ", (void *) handle, keyName (parentKey), keyString (parentKey));
```
[the remark this answers is not in the document]
Comment (kodebach) [reply src/plugins/tracer/tracer.c:128]: Yup, just forgot to change it

**Comment by kodebach** [comment]
> There was a build error on centos-stream-8 with the jni plugin

Yes, saw that. No idea why. It looks a bit as if we compiled with a newer Java version than we run the test with. I'll investigate further...

**Comment by mpranj** [comment]
> It looks a bit as if we compiled with a newer Java version than we run the test with.

I think you're right. This also comes up:

```
-- Exclude binding jna because At least Java 11 is required, but 1.8.0_362 was found
```
.. even though the dockerfile has `java-11-openjdk-devel` installed.

Maybe `alternatives --auto java && alternatives --auto javac` does not set the correct version automatically.

**Comment by kodebach** [comment]
Yup it's definitely the Java Version:

```
Status: Downloaded newer image for hub-public.libelektra.org/build-elektra-centos-stream-8:202303-e90faa200a03efcfb4ed11d3e4fca87c28cc71c1c8cd21f25e5b21c41610df67
[jenkins@57159c5544ab /]$ java -version
openjdk version "1.8.0_362"
OpenJDK Runtime Environment (build 1.8.0_362-b08)
OpenJDK 64-Bit Server VM (build 25.362-b08, mixed mode)
[jenkins@57159c5544ab /]$ javac -version
javac 11.0.18-ea
```

File: tests/shell/check_kdb_internal_check.sh [hunk 2]
```diff
@@ -22,61 +22,30 @@ if contains "$KDB" "full" || contains "$KDB" "static"; then
...
-		;;
-	"timeofday")
-		# output on open/close
-		continue
-		;;
-	"counter")
-		# output on open/close
-		continue
-		;;
-	"xfconf")
```
Comment (mpranj) [inline tests/shell/check_kdb_internal_check.sh:49]: From the builds it looks like we will have to disable at least `xfconf`.

Comment (kodebach) [reply tests/shell/check_kdb_internal_check.sh:49]: Ah okay, so it doesn't work during packaging. I'll see if I can disable it only at that time somehow...

**Comment by mpranj** [comment]
Thank you so much for the fixes!

**Pull request merged**